advanced_entity_reference_formatter.info.yml

```diff
@@ -0,0 +1,5 @@
+name: 'Advanced entity reference formatter'
+type: module
+description: 'Provides advanced formatter for entity reference fields, which allows to select view modes for target bundles or items delta separately.'
```

Comment:
This is just a small text suggestion:
"Provides advanced formatter for entity reference fields, allowing to select different view modes per target bundle or  delta item.". For both texts: This one, and text into the "README.md".